frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -742,6 +743,28 @@ export async function getDatasets(
   return datasets;
 }
 
+export async function getJobs(): Promise<Array<APIJob>> {
+  const jobs = await Request.receiveJSON("/api/jobs");
+  assertResponseLimit(jobs);
+  return jobs.map(job => ({
+    id: job.id,
+    type: job.command,
+    datasetName: job.commandArgs.kwargs.dataset_name,
+    state: _.capitalize(job.celeryInfo.state.toLowerCase()),
```

Comment:
I think, it would be better to not call the `capitalize` function here. It is only needed for formatting purposes during rendering, which is why I wouldn't change the internal representation here (usually, it's nice to have the server's and the client's representation as similar as possible).

Instead, you can use the `render={(__, job: APIJob) => ...` prop in the State-`<Column />` in the table to do the capitalization there.

Reply:
Should the toLowerCase consequently happen there, too?

Reply:
Ah, yes, sorry, I forgot to mention this. Both should be moved so that this line is only `state: job.celeryInfo.state`.